doc/how-to/how-to-setup-custom-subdomains.md

```diff
@@ -0,0 +1,93 @@
+    - Certificates can be provided as individual files, or in an archive format (`.pfx`, `.pkcs12`, `.p12`)
+
+4. **PlanX** - Format certificates if provided with PKCS #12
+    ```shell
+    openssl pkcs12 -nocerts -nodes -in <FILENAME> -out council.key
+    openssl pkcs12 -nokeys -in <FILENAME> -out council.cert
+    ```
+    
+    The `.cert` file might contain the certificate chain inside it, so please separate the first certificate in the file (the certificate body) from the rest of the certificates in the file (the certificate chain).
+
+5. **PlanX** - Validate certificates manually in AWS Console (optional but recommended)
+  
+    To avoid slow round trip loops of Pizza -> Staging -> Production, certificates can be manually imported via the AWS Console to check their validity before proceeding.
+
+    In order to do this -
+
+      - Log into AWS staging 
+      - Navigate to AWS Certificate Manager (ACM)
+      - Change to region to us-east-1 (N.Virginia) (required for CloudFront)
+      - Import certificate (copy cert, key and chain into form)
+      - Check the following after import - 
+        - Subdomain is correct ✅
+        - Key algorithm is RSA 2048 ✅
+        - Navigate to AWS CloudFront > Create new distribution. The certificate is listed in the dropdown for "Custom SSL certificate" ✅
+      - If the above checks pass, proceed to next step
+
+6. **PlanX** - Add certificates to AWS infrastructure
+
+    Certificates and keys are added to our infrastructure as Pulumi secrets which are then read to generate an ACM record and CloudFront distribution for the custom domain when a deployment is made to Production.
+
+    1. Add the team to the `CUSTOM_DOMAINS` array in `infrastructure/application/index.ts`
+
+    2. Add secrets to Pulumi
+
+        The following secrets need to be added to the Pulumi application layer - `ssl-{team}-key`, `ssl-{team}-cert` and `ssl-{team}-chain` (optional). These secrets are only required for the production stack.
+
+        The provided files, or output of step 4, can be piped directly into these variables.
+
+        ```bash
+        cd infrastructure/application
+
+        cat council.key | pulumi config set ssl-{team}-key --stack production --secret
+
+        cat council.cert | pulumi config set ssl-{team}-cert --stack production --secret
+
+        cat chain.cert | pulumi config set ssl-{team}-chain --stack production --secret
+        ```
+
+        To validate that you have successfully added the secrets to Pulumi, check `infrastructure/application/Pulumi.production.yaml` which will list the above secrets for the team in an encrypted format.
+
+7. **PlanX** - Deploy to production in order to create ACM records and CloudFront CDN
+
+8. **PlanX** - Provide IT team with domain of CloudFront CDN. This will be in format `<abc123>.cloudfront.net` and is listed on the CloudFront dashboard (via AWS Console).
+  
+9. **IT Team** - Create CNAME DNS record pointing at `<abc123>.cloudfront.net`
```

Comment:
I guess the reason this isn't done asynchronously and up-front is that we haven't created the CloudFront distribution until step 7.
Just an idea - would it be possible to create the distribution up front (alongside step 3?) and then edit it to include certs afterwards? If so, this may enable less back and forth and allow us to test the end-to-end (perhaps with a holding page instead of the application).